src/invocation/ClusterService.ts

```diff
@@ -262,7 +271,9 @@ export class ClusterService extends EventEmitter {
     }
 
     private handleMemberAttributeChange(uuid: string, key: string, operationType: number, value: string): void {
-        this.emit(EMIT_ATTRIBUTE_CHANGE, uuid, key, ATTRIBUTE_CHANGE[operationType], value);
+        const member = this.getMember(uuid);
+        const memberAttributeEvent = new MemberAttributeEvent(member, key, ATTRIBUTE_CHANGE[operationType], value);
```

Comment:
If we change operationType in MemberAttributeEvent to an enum, we can get rid of ugly `ATTRIBUTE_CHANGE` object, too.